### PR TITLE
Introduce the examples from Norway & change Covered from boolean to enumeration

### LINKED ISSUE #682: Covered is now boolean and an enumeration make it uniform?

### PR DESCRIPTION
**Pull request by skinkie**
Fix #682

### CODE REVIEW

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentParking_version.xsd [hunk 1]
```diff
@@ -127,9 +127,9 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>whether shelter is enclosed.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
-			<xsd:element name="Covered" type="xsd:boolean" minOccurs="0">
+			<xsd:element name="Covered" type="CoveredEnumeration" minOccurs="0">
 				<xsd:annotation>
-					<xsd:documentation>Whether storage is covered.</xsd:documentation>
+					<xsd:documentation>Whether the component is indoors or outdoors.</xsd:documentation>
```
Comment (trurlurl) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentParking_version.xsd:132]:
```suggestion
					<xsd:documentation>Weather protection.</xsd:documentation>
```

Comment (skinkie) [reply xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentParking_version.xsd:132]: Your suggestion is not English.

Comment (trurlurl) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentParking_version.xsd:132]:
```suggestion
					<xsd:documentation>Whether the parking space is protected from weather.</xsd:documentation>
```

Comment (trurlurl) [reply xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentParking_version.xsd:132]: Not really a sentence either in my opinion, but at least a full half-sentence now :-) Please feel free adding another half or keeping the initial solution.

**Review by trurlurl — APPROVED** [review]

**Review by ue71603 — APPROVED** [review]

**Comment by skinkie** [comment]
@ue71603 want to discuss tomorrow or merge

**Comment by ue71603** [comment]
@skinkie if we get a go from ENTUR, then i would say merge.

**Comment by ue71603** [comment]
@skinkie  if ok for you, then merge.

**Comment by skinkie** [comment]
I recall @trurlurl mentioned we needed to add true / false. I don't know why it is not in this one.

**Comment by ue71603** [comment]
@skinkie  I think we discussed it. However, it is not in here. So where do we go... multiple booleans? enumeration ? list? @trurlurl

**Comment by trurlurl** [comment]
> @skinkie I think we discussed it. However, it is not in here. So where do we go... multiple booleans? enumeration ? list? @trurlurl

The idea was to add "True" and "False" to *CoveredEnumeration*.

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentParking_version.xsd [hunk 2]
```diff
@@ -127,9 +127,9 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>whether shelter is enclosed.</xsd:documentation>
```
Comment (trurlurl) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentParking_version.xsd:127]:
```suggestion
					<xsd:documentation>Whether shelter is enclosed.</xsd:documentation>
```

**Comment by ue71603** [comment]
> > @skinkie I think we discussed it. However, it is not in here. So where do we go... multiple booleans? enumeration ? list? @trurlurl
> 
> The idea was to add "True" and "False" to _CoveredEnumeration_.

added now

File: xsd/netex_framework/netex_reusableComponents/netex_environment_support.xsd [hunk 3]
```diff
@@ -449,6 +449,16 @@ Rail transport, Roads and Road transport
...
+			<xsd:enumeration value="true">
+				<xsd:annotation>
+					<xsd:documentation>Value to be used, if covered is used as a "boolean". Equals covered. +v2.0</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="false">
+				<xsd:annotation>
+					<xsd:documentation>Value to be used, if covered is used as a "boolean". Equals outdoors. +v2.0</xsd:documentation>
+				</xsd:annotation>
+			</xsd:enumeration>
```
Comment (trurlurl) [inline xsd/netex_framework/netex_reusableComponents/netex_environment_support.xsd:461]: Should there be noted that this usage is deprecated?

**Comment by skinkie** [comment]
The strange thing is: I recall I added it.

**Comment by skinkie** [comment]
> > The strange thing is: I recall I added it.
> 
> That's not so strange. When planning and starting to do something, the projected imagination of having achieved the goal may remain as persistent memory if the execution of the action sequence and the accompanying attentional processes has been completely interrupted and abandoned because of some external event. What remains is a false memory of having achieved the goal. At least, that's a theory how things work that I recall having either found or elaborated myself - but strangely I don't remember any more details about it...

Lets put it this way, when you mentioned it in the last working group. I _directly_ did it. So I wonder where it ended up.

**Comment by ue71603** [comment]
perhaps it was undone by #681?

**Comment by skinkie** [comment]
I am not crazy.

https://github.com/NeTEx-CEN/NeTEx/pull/683/commits/cd00ea6ae716d1c9fec0940492d2f25cb846794d

**Comment by trurlurl** [comment]
> I am not crazy.
> 
> [cd00ea6](https://github.com/NeTEx-CEN/NeTEx/commit/cd00ea6ae716d1c9fec0940492d2f25cb846794d)

@skinkie I don't think anybody ever suspected you being crazy (except perhaps yourself) - I would rather have thought that another theoretical explanation would have applied in that case: that it was the universe who simply forgot the change you had done.

File: xsd/netex_framework/netex_reusableComponents/netex_environment_support.xsd (hunk 3 above)
Comment (ue71603) [reply xsd/netex_framework/netex_reusableComponents/netex_environment_support.xsd:461]: done

**Pull request merged**